lib/AccessSystem/Form/Person.pm

```diff
@@ -57,6 +60,42 @@ sub validate {
     }
 }
 
+# https://metacpan.org/pod/HTML::FormHandler::Manual::Cookbook#Handle-extra-database-fields ???
+
+around update_model => sub {
+    my $orig = shift;
+    my $self = shift;
+
+    my $item   = $self->item;
+    my $source = $self->source;
+
+    if ($self->value->{plate_reg}) {
+        $self->plate_reg(delete $self->value->{plate_reg});
+    }
+
+    $self->$orig(@_);
+
+    if ($self->plate_reg) {
+        my $pr = $self->plate_reg;
+        $pr =~ s/\W//g;
+        $pr = uc($pr);
+        $pr =~ s/_//g;
+        $self->item->update_or_create_related('vehicles' => { plate_reg => $pr });
+    }
+    
+    # {
```

Comment:
Is this deleteable dead code?

Reply:
Kind of? I'd like to do it "correctly" at some point..